src/playback-watcher.js

```diff
@@ -51,7 +51,9 @@ export default class PlaybackWatcher {
     this.tech_.on('seekablechanged', fixesBadSeeksHandler);
     this.tech_.on('waiting', waitingHandler);
     this.tech_.on(timerCancelEvents, cancelTimerHandler);
-    this.monitorCurrentTime_();
+    if (this.tech_.hasStarted_) {
```

Comment:
If the player has not started yet, but starts later, will PlaybackWatcher end up running?